examples/IRMQTTServer/IRMQTTServer.ino

```diff
@@ -1000,6 +1000,7 @@ String htmlHeader(const String title, const String h1_text) {
   html += title;
   html += F("</title><meta http-equiv=\"Content-Type\" "
             "content=\"text/html;charset=utf-8\">"
+            "<meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=5.0\">"
```

Comment:
Linter error:
  `examples/IRMQTTServer/IRMQTTServer.ino:1003:  Lines should be <= 80 characters long  [whitespace/line_length] [2]`